src/cypress/integration/examples/actions.spec.js

```diff
@@ -0,0 +1,299 @@
+/// <reference types="cypress" />
+
```

Comment:
Very nitpicky thing but in the future you may want to consider deleting the tests in the example folder so that your code is easier to review (and so that you're not committing unrelated code)